GOVERNANCE.md

```diff
@@ -98,19 +106,43 @@ There is no special process for changing this document. Submit a PR and ask for
 
 Please also make a note in the change log under [`# Governance history`](#governance-history)
 
+# Finances and Funding
+
+There is currently no mechanism for the data.table project to receive funding as an entity.  
+
+Funding support for this project therefore may come in two forms:
+
+## Individual external funding
+
+Any individual developer or community member of data.table may apply for and receive funding for their work on the project.  Individuals or groups seeking funding support are strongly encouraged to consult directly with the data.table Project Members (by initiating an Issue on GitHub) to ensure funds are used meaningfully. Formally, however, decisions about use of funds are governed by the individual grantee(s) and their contract with the funding agency. 
+
+There is no guarantee that funded work will be incorporated into the data.table package; any contributions, whether funded or unfunded, are subject to the same review process as outlined above.
+
+## Direct donations
+
+Direct donations to the project may be made via GitHub Sponsorships, which allow individuals to fund a specific developer.  Those wishing to support the project in general, rather than a specific developer, should direct their donations to the current CRAN Maintainer.
+
+## Decision-making for future opportunities
+
+We here outline a procedure for disbursing funds, should this project in the future become a directly fundable entity (e.g. an LLC or a subsidiary of an umbrella LLC).
+
+Funds acquired by the data.table project will be at the discretion of the **committers**, defined as above.  The **CRAN Maintainer** will have authority to make final decisions in the event that no consensus is reached among committers prior to deadlines for use of funds, and will be responsible for disbursement logistics.
+
 # Code of conduct
 
+The full Code of Conduct can be found [here](CODE_OF_CONDUCT.md), including details for reporting violations.
+
+## Pledge
+
 As contributors of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
 We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, etc.
```

Comment:
since this part is redundant with the newly added CODE_OF_CONDUCT.md, I would suggest removing it here, what you think?

Reply:
I agree since we specifically reference the CoC in this doc already.